documentation/commands.md

```diff
@@ -197,22 +233,8 @@ print out above the command bar.
 
 _Syntax -_ `AAL123 sr`
 
-### ~~Fix~~
-~~_Aliases -_ `f` / `fix` / `track`~~
-
-~~_Syntax -_ `AAL123 f [fixname]`~~
-*_This command has been deprecated_*
-
-### Proceed Direct
-_Aliases -_ `direct` / `pd` / `dct`
-
-_Information -_ This command instructs the aircraft to go direct to a
-navigational fix, taking a shortcut. For example, if an aircraft is flying
-to fixes [A, B, C], issuing the command "pd B" will cause the aircraft
-to go to B, then C. After flying past the last fix, the aircraft will
-continue flying straight.
-
-_Syntax -_ `AAL123 pd [fixname]`
+## Aircraft Control commands
+These commands modify the aircraft but are not limited to departures or arrivals.
```

Comment:
These commands control the three most basic ways we can control aircraft, collectively called the "basic control instructions".